internal/services/ingest.go

```diff
@@ -171,7 +171,7 @@ func (m *ingestService) DeprecatedRun(ctx context.Context, startLedger uint32, e
 			m.metricsService.ObserveIngestionDuration(totalIngestionPrometheusLabel, time.Since(start).Seconds())
 
 			// immediately trigger the next ingestion the wallet-backend is behind the RPC's latest ledger
-			if resp.LatestLedger-ingestLedger > 1 {
+			if resp.LatestLedger-ingestLedger > 1 && len(manualTriggerChannel) == 0 {
```

Comment:
I am a little confused so can you revisit why the `manualTriggerChannel` is needed? Wouldnt the `rpcHeartbeatChannel` keep the loop running and we would catchup to the tip if we fall behind?